From 71e1dbd378ac98783d0ec1c155f256fb800b2632 Mon Sep 17 00:00:00 2001
From: Enrico Scholz <enrico.scholz@sigma-chemnitz.de>
Date: Thu, 18 May 2017 18:17:01 +0200
Subject: [PATCH] removed bb.data.setVar() usage

fixes

         except bb.parse.SkipRecipe:
    >        bb.data.setVar("__SKIPPED", True, d)
             if include == 0:
AttributeError: module 'bb.data' has no attribute 'setVar'

Signed-off-by: Enrico Scholz <enrico.scholz@sigma-chemnitz.de>
---
 lib/bb/parse/parse_py/BBHandler.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/bb/parse/parse_py/BBHandler.py b/lib/bb/parse/parse_py/BBHandler.py
index fe918a4..f89ad24 100644
--- a/lib/bb/parse/parse_py/BBHandler.py
+++ b/lib/bb/parse/parse_py/BBHandler.py
@@ -144,7 +144,7 @@ def handle(fn, d, include):
     try:
         statements.eval(d)
     except bb.parse.SkipRecipe:
-        bb.data.setVar("__SKIPPED", True, d)
+        d.setVar("__SKIPPED", True)
         if include == 0:
             return { "" : d }

--
2.9.4
